ledger/participant-integration-api/src/main/scala/platform/indexer/JdbcIndexer.scala

```diff
@@ -20,35 +20,26 @@ import com.daml.platform.store.DbType.{
 }
 import com.daml.platform.store.appendonlydao.events.{CompressionStrategy, LfValueTranslation}
 import com.daml.platform.store.backend.DataSourceStorageBackend.DataSourceConfig
-import com.daml.platform.store.backend.{
-  DataSourceStorageBackend,
-  ResetStorageBackend,
-  StorageBackendFactory,
-}
+import com.daml.platform.store.backend.StorageBackendFactory
 import com.daml.platform.store.backend.postgresql.PostgresDataSourceConfig
 import com.daml.platform.store.{DbType, LfValueTranslationCache}
 
-import scala.concurrent.{ExecutionContext, Future}
-import scala.util.Using
+import scala.concurrent.Future
 
 object JdbcIndexer {
   private[daml] final class Factory(
       config: IndexerConfig,
       readService: state.ReadService,
-      servicesExecutionContext: ExecutionContext,
       metrics: Metrics,
       lfValueTranslationCache: LfValueTranslationCache.Cache,
   )(implicit materializer: Materializer) {
 
-    def initialized(
-        resetSchema: Boolean = false
-    )(implicit loggingContext: LoggingContext): ResourceOwner[Indexer] = {
+    def initialized(implicit loggingContext: LoggingContext): ResourceOwner[Indexer] = {
```

Comment:
```suggestion
    def initialized()(implicit loggingContext: LoggingContext): ResourceOwner[Indexer] = {
```

@mziolekda @rautenrieth-da This should make the change in indexer-benchmark unnecesary.

Reply:
:+1: